stl/src/vector_algorithms.cpp

```diff
@@ -450,6 +451,354 @@ __declspec(noalias) void __cdecl __std_reverse_copy_trivially_copyable_8(
         static_cast<unsigned long long*>(_Dest));
 }
 
+} // extern "C"
+
+template <class _Ty>
+static const void* _Find_trivial_unsized_fallback(const void* _First, _Ty _Val) {
+    auto _Ptr = static_cast<const _Ty*>(_First);
+    while (*_Ptr != _Val) {
+        ++_Ptr;
+    }
+    return _Ptr;
+}
+
+template <class _Ty>
+static const void* _Find_trivial_tail(const void* _First, const void* _Last, _Ty _Val) {
+    auto _Ptr = static_cast<const _Ty*>(_First);
+    while (_Ptr != _Last && *_Ptr != _Val) {
+        ++_Ptr;
+    }
+    return _Ptr;
+}
+
+template <class _Ty>
+__declspec(noalias) size_t _Count_trivial_tail(const void* _First, const void* _Last, size_t _Current, _Ty _Val) {

```

Comment:
(Going to push changes for this, but wanted to record why - I had a comment for this earlier but somehow lost it.)

Some but not all of the functions outside of `extern "C"` are `static`. Now that this is gaining types like `struct _Find_traits_1` (which can't be `static`), I think we should change to using an unnamed namespace for all support machinery here. As this is injected into the import lib, we should be maximally clean regarding linkage.